python/tvm/relay/backend/interpreter.py

```diff
@@ -73,6 +73,51 @@ def _arg_to_ast(arg):
 
 class Executor(object):
     """An abstract interface for executing Relay programs."""
+
+    def _convert_args(self, expr, args, kwargs):
+        """
+        Convert the combination of args and kwargs into
+        a sequence of arguments that can be passed to
+        a Relay evaluator.
```

Comment:
Should we explicitly say in the comment that args are always added first before kwargs?
Otherwise, lgtm

Reply:
Yeah I will update the comment, and then we can merge? the rough algorithm is all positional arguments are passed and then the remaining positions are filled in by kwargs.